.travis.yml

```diff
@@ -38,6 +38,8 @@ matrix:
         - PG_PORT=5433
         - PG_HOST=localhost
         - REDIS_CLUSTER_IP_TEST=localhost
+        - PG_DATABASE=brave
```

Comment:
After your merge, the `PG_DATABASE` value is in here twice.

Reply:
Will fix that, thanks!